### PR TITLE
{bio}[foss/2021a] batchgenerators v0.25, HD-BET v20220318 w/ Python 3.9.5

### PR DESCRIPTION
**Pull request by lexming**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by lexming** [comment]
@boegelbot: please test @ generoso

File: easybuild/easyconfigs/b/batchgenerators/batchgenerators-0.25-foss-2021a.eb [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+dependencies = [
+    ('Python', '3.9.5'),
+    ('SciPy-bundle', '2021.05'),
+    ('scikit-image', '0.18.3'),
+    ('scikit-learn', '0.24.2'),
+]
+
+# remove requirement on Pillow and unittest2, which are not actually used
+preinstallopts = "sed -i '13d;19d' setup.py &&"
+
```
Comment (Micket) [inline easybuild/easyconfigs/b/batchgenerators/batchgenerators-0.25-foss-2021a.eb:25]: I think this can add
runtest = 'pytest'

Comment (lexming) [reply easybuild/easyconfigs/b/batchgenerators/batchgenerators-0.25-foss-2021a.eb:25]: the tarball from PyPI does not have the tests :cry:

**Review by Micket — APPROVED** [review]
lgtm

**Comment by Micket** [comment]
Test report by @Micket
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in total)
vera-c1 - Linux Rocky Linux 8.6, x86_64, Intel Xeon Processor (Skylake), Python 3.6.8
See https://gist.github.com/Micket/6a9c28defd2540938c4143db70433eb9 for a full test report.

**Comment by Micket** [comment]
Going in, thanks @lexming!

**Pull request merged**